app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -226,9 +314,32 @@ fun EditActivityScreen(
                   Icons.Outlined.Delete,
                   contentDescription = "add a new activity",
               )
-              Text("Create")
+              Text("Delete")
```

Comment:
very important fix ! I have had an issue with this in the previous version, where I deleted an activity instead of creating it